pkg/api/types.go

```diff
@@ -2427,7 +2427,7 @@ type SerializedReference struct {
 type EventSource struct {
 	// Component from which the event is generated.
 	Component string `json:"component,omitempty"`
-	// Host name on which the event is generated.
+	// Node name on which the event is generated.
```

Comment:
Oh I only searched inside pkg/ and Host is set in cmd/. I agree it should actually be termed as node name.